OneSignalSDK/onesignal/src/main/AndroidManifest.xml

```diff
@@ -64,8 +64,9 @@
         <receiver android:name="com.onesignal.NotificationOpenedReceiver" />
 
         <service android:name="com.onesignal.GcmIntentService" />
-        <service android:name="com.onesignal.GcmIntentJobService"
-                 android:permission="android.permission.BIND_JOB_SERVICE" />
+
+        <service android:name="com.onesignal.GcmJobIntentService"
```

Comment:
I do like the renaming here but unfortunately for those who use our .jar instead of our .aar this would be a breaking change. Since they would need to update their `AndroidManifest.xml` manually in the .jar case. Can we put back the name?

Reply:
Oh good catch!